bokeh/models/actions.py

```diff
@@ -0,0 +1,17 @@
+""" Client-side interactivity. """
+
+from __future__ import absolute_import
+
+from ..plot_object import PlotObject
+from ..properties import String
+
+class Action(PlotObject):
+    """ Base class for interactive actions. """
+
+class OpenURL(Action):
+    """ Open a URL in a new tab or window (browser dependent). """
+
+    url = String("http://", help="""
+    The URL to direct the web browser to. This can be a template string,
+    which will be formatted with data from the data source.
+    """)
```

Comment:
@mattpap This was what I was thinking of - thank you for reading my late night mind!

@mattpap this is awesome - thanks! @bryevdv when I was messing around with this last night, the extra actions I immediately added which seem like they might be easy out-of-the-box wins:
* ConsoleLog - just for debugging, logs a value in the console
* DivText (or maybe even DivHtml - infact that would definitely be better) - takes two parameters: `selector` (a jquery selector) to update, and `value` some html that would become the inner html of the elected element(s).